orc8r/gateway/c/common/sentry/SentryWrapper.cpp

```diff
@@ -84,6 +85,7 @@ std::string get_snowflake() {
 }
 
 void initialize_sentry(
+    char node_name[50]
```

Comment:
For portability we can use `HOST_NAME_MAX` constant for specifying the node_name length

Reply:
yep make sense. thanks !!!